### PR TITLE
Changes in SecurityController.php and login_content.html.twig

### PR DESCRIPTION
**Pull request by sgb004**
When you try to use a common design in your forms and you try to use form_row, form_widget or form_label in login form (login_content.html.twig) you can't use it because controller don't send a form view, with these changes you can use form_row, form_widget, form_label, etc., and you can avoid repeat html.

### CODE REVIEW

File: Controller/SecurityController.php [hunk 1]
```diff
@@ -53,7 +53,16 @@ public function loginAction(Request $request)
             ? $this->get('security.csrf.token_manager')->getToken('authenticate')->getValue()
             : null;
 
+         $form = $this->createFormBuilder()
```
Comment (core23) [inline Controller/SecurityController.php]: Can you create a FormType for this?

File: Resources/views/Security/login_content.html.twig [hunk 2]
```diff
@@ -9,14 +9,11 @@
         <input type="hidden" name="_csrf_token" value="{{ csrf_token }}" />
     {% endif %}
 
-    <label for="username">{{ 'security.login.username'|trans }}</label>
-    <input type="text" id="username" name="_username" value="{{ last_username }}" required="required" />
+    {{ form_row(form._username, {'full_name': '_username', 'label': 'security.login.username'|trans}) }}
```
Comment (core23) [inline Resources/views/Security/login_content.html.twig]: Can you put the attributes (full_name, label) to the PHP part? All other template do the same.

File: Controller/SecurityController.php [hunk 3]
```diff
@@ -53,7 +57,16 @@ public function loginAction(Request $request)
...
+            ->add('_username', TextType::class, array('data' => $lastUsername))
+            ->add('_password', PasswordType::class)
+            ->add('_remember_me', CheckboxType::class, array('required' => false))
+            ->add('_submit', SubmitType::class)
```
Comment (XWB) [inline Controller/SecurityController.php]: As mentioned in the other PR, this won't work on Symfony 2.7.

File: Controller/SecurityController.php (hunk 1 above)
Comment (sgb004) [inline Controller/SecurityController.php]: Yes of curse, I'll try to add a FormType to SecurityController.php

File: Resources/views/Security/login_content.html.twig (hunk 2 above)
Comment (sgb004) [inline Resources/views/Security/login_content.html.twig]: I tried to put full_name in Controller, but it requires extra innecesary code, and text's label is more easy and quickly to edit if it is in the login_content.html.twig

File: Controller/SecurityController.php (hunk 3 above)
Comment (sgb004) [inline Controller/SecurityController.php]: Yes, you mentioned in the other PR, I need to add a FormType to add compatibility, thanks!

**Comment by XWB** [comment]
@sgb004 Any updates?

**Comment by sgb004** [comment]
Sorry @XWB, I have an error: Could not load type "FOS\UserBundle\Form\Type\SecurityFormType" in Symfony 2.7

**Comment by XWB** [comment]
Closing this PR as it is outdated.

**Pull request closed without merging**